micro-transactions/src/test/java/com/oath/micro/server/transactions/TransactionFlowTest.java

```diff
@@ -53,8 +53,8 @@ public void errorHandlingGeneral() {
 										.map(this::load)
 										.map(this::error)
 										.execute(10)
-										.toFailedOptional()
-										.get();
+                                        .toFailedOption()
```

Comment:
minor: is the new indentation of some lines in this file intentional

Reply:
reformatted